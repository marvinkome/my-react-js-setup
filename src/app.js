import React from 'react';
import 'materialize-css/dist/css/materialize.css';

export const App = () => <p>hello</p>;