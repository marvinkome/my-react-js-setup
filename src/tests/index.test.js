import React from 'react'
import { shallow } from 'enzyme'
import { App } from '../app'
import { wrap } from 'module'

describe('App tests', () => {
    it('renders correctly', () => {
        const app = shallow(<App />)
        expect(app).toMatchSnapshot()
    })
})
