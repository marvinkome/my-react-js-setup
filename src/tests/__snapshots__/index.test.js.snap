// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "hello",
    },
    "ref": null,
    "rendered": "hello",
    "type": "p",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "hello",
      },
      "ref": null,
      "rendered": "hello",
      "type": "p",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
